src/classes/TDTM_DMLgt200_TEST.cls

```diff
@@ -0,0 +1,465 @@
+        if (strTestOnly != '*' && strTestOnly != 'test_contacts_with_primary_affiliations') return;
+
+        npe5__Affiliations_Settings__c affiliationsSettingsForTests = UTIL_CustomSettingsFacade.getAffiliationsSettingsForTests(
+                new npe5__Affiliations_Settings__c(npe5__Automatic_Affiliation_Creation_Turned_On__c = true));
+
+        Id orgRTId = [SELECT Id FROM RecordType WHERE SobjectType = 'Account' AND DeveloperName = 'Organization' AND IsActive = True LIMIT 1].Id;
+        Account orgAcct = new Account(Name='TestOrganization', RecordTypeId = orgRTId);
+        insert orgAcct;
+
+        Test.startTest();
+
+        // Create more than 200 Contact records and then validate that all of the Household Accounts were created
+        // as well as all the related Affiliation recorsd
+        List<Contact> contacts = new List<Contact>();
+        for (Integer n=0; n<insertCnt; n++) {
+            contacts.add(new Contact(
+                    FirstName = 'TEST' + n,
+                    LastName = 'TestContact# ' + n,
+                    Title = 'TestTitle',
+                    Primary_Affiliation__c = orgAcct.Id
+            ));
+        }
+        database.insert(contacts);
+        Test.stopTest();
+        system.AssertEquals(insertCnt, [SELECT Count() FROM Account WHERE RecordType.DeveloperName = 'HH_Account'],
+                'There should be exactly ' + insertCnt.format() + ' Household Account records');
+        system.AssertEquals(insertCnt, [SELECT Count() FROM npe5__Affiliation__c WHERE npe5__Organization__c = :orgAcct.Id],
+                'There should be exactly ' + insertCnt.format() + ' Affiliations records');
+        system.AssertEquals(insertCnt, [SELECT Count() FROM Contact WHERE Primary_Affiliation__c = :orgAcct.Id],
+                'There should be exactly ' + insertCnt.format() + ' Contacts with a Primary Affilition');
+    }
+
+    /*********************************************************************************************************
+    * @description Test creating 200+ Contacts with Relationships
+    */
+    @isTest
+    static void test_contacts_with_relationships() {
+        if (strTestOnly != '*' && strTestOnly != 'test_contacts_with_relationships') return;
+
+        Test.startTest();
+
+        // Create more than 200 Contact records and then validate that all of the Household Accounts were created
+        List<Contact> contacts = new List<Contact>();
+        for (Integer n=0; n<insertCnt; n++) {
+            contacts.add(new Contact(
+                    FirstName = 'TEST' + n,
+                    LastName = 'TestContact# ' + n,
+                    Title = 'TestTitle'
+            ));
+        }
+        database.insert(contacts);
+        system.AssertEquals(insertCnt, [SELECT Count() FROM Account WHERE RecordType.DeveloperName = 'HH_Account'],
+                'There should be exactly ' + insertCnt.format() + ' Household Account records');
```

Comment:
nix the record type.  you can find them by querying on Account.
cHHAcc = [select Count() from Account 
                where npe01__SYSTEM_AccountType__c = :CAO_Constants.HH_ACCOUNT_TYPE]